spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -759,6 +763,37 @@ def set_max_results(self, value=None):
         else:
             self.set_conf('max_results', value)
 
+    def show_error(self, error_msg, exclude):
+        """Show a regexp error message with an icon."""
+        self._show_icon_message(extra_info=error_msg, exclude=exclude)
+
+    def _show_icon_message(self, extra_info=None, exclude=False):
+        """
+        Show a message to users with an icon when there's an error in the passed regexp.
+
+        Parameters
+        ----------
+        extra_info:
+            Extra info to add to the icon's tooltip.
+        """
+        tooltip = self.REGEX_ERROR
+        icon = self.error_icon
+
+        if extra_info:
+            tooltip = tooltip + ': ' + extra_info
+        if exclude:
+            self.messages_exclude_action.setIcon(icon)
+            self.messages_exclude_action.setToolTip(tooltip)
+            self.messages_exclude_action.setVisible(True)
+            tooltip_search = _("Regular expression error in exclude")
```

Comment:
```suggestion

            tooltip_search = _("Regular expression error in Exclude field")
```

Improve text and blank line for clarity.